### PR TITLE
Instructions to update the Audio HAT firmware

### PR DESCRIPTION
**Pull request by aallan**
Add instructions to update the Audio HAT firmware.

### CODE REVIEW

File: documentation/asciidoc/accessories/audio/update-firmware.adoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+=== EEPROM Programming
+
+Once the write-protect line has been pulled down, the EEPROM can be programmed. 
+
+You should first install the utilites and then run the programmer. Open up a terminal window and type the following:
+
+----
+$ sudo apt install iqaudio-utils
+$ sudo iqaudio-program
```
Comment (XECDesign) [inline documentation/asciidoc/accessories/audio/update-firmware.adoc:23]: That should be
```
sudo apt update
sudo apt install rpi-audio-utils
sudo rpi-audio-flash
```

Comment (aallan) [reply documentation/asciidoc/accessories/audio/update-firmware.adoc:24]: Fixed in [e26881d](https://github.com/raspberrypi/documentation/pull/2767/commits/e26881da693fe896b79852c9a175cbe3a032fc93)

Comment (XECDesign) [reply documentation/asciidoc/accessories/audio/update-firmware.adoc:24]: I see that 'sudo apt update' was added, but the package and script names are still the `iqaudio` version.

Comment (aallan) [reply documentation/asciidoc/accessories/audio/update-firmware.adoc:24]: Oops.

Comment (aallan) [reply documentation/asciidoc/accessories/audio/update-firmware.adoc]: Fixed properly in [b57861e](https://github.com/raspberrypi/documentation/pull/2767/commits/b57861e6213820787f603724bb3e72e5aa205095)

**Pull request merged**